syncano_sync/__init__.py

```diff
@@ -0,0 +1,15 @@
+import logging
+from contextlib import contextmanager
+
+
+LOG = logging.getLogger('syncano-sync')
+SYNCANO_LOG = logging.getLogger('syncano')
+
+
+@contextmanager
+def mute_log(logger=None):
```

Comment:
It issues warnings when saving objects with read_only fields set, fetched from syncano. I reported this issue for @opalczynski :)